assets/data/responsive-widgets.json

```diff
@@ -1 +1 @@
-{"accordion":true,"alert":true,"icon-box":true,"icon-list":true,"image-box":true,"image-gallery":true,"progress":true,"star-rating":true,"tabs":true,"toggle":true}
+{"accordion":true,"alert":true,"icon-box":true,"icon-list":true,"image-box":true,"image-gallery":true,"link-in-bio":true,"progress":true,"star-rating":true,"tabs":true,"toggle":true}
```

Comment:
Please remove the changes to this file.

Reply:
Removing. I think this was automatically generated though.

Reply:
done